src/main/java/net/mcreator/element/converter/ConverterRegistry.java

```diff
@@ -43,6 +44,7 @@
 		put(ModElementType.BIOME, Collections.singletonList(new BiomeSpawnListConverter()));
 		put(ModElementType.OVERLAY, Collections.singletonList(new OverlayCoordinateConverter()));
 		put(ModElementType.BIOME, Collections.singletonList(new BiomeDefaultFeaturesConverter()));
+		put(ModElementType.BIOME, Collections.singletonList(new BiomeTreeConverter()));

```

Comment:
You can't put twice, this will override the previous entry. Do not ignore warnings by IDEA.